src/to-car.ts

```diff
@@ -22,7 +22,7 @@ import globSource from 'ipfs-utils/src/files/glob-source'
 import { WriterChannel } from '@ipld/car/api'
 
 export async function packFileToCarFs({ input, output }: { input: string | Iterable<string> | AsyncIterable<string>, output?: string }) {
-  const location = output || `${os.tmpdir()}/${(parseInt(String(Math.random() * 1e9))).toString(36) + Date.now()}`
+  const location = output || `${os.tmpdir()}/${(parseInt(String(Math.random() * 1e9), 2)).toString(36) + Date.now()}`
```

Comment:
this is just temp file writing! Unless there is a good reason to, let's calm down these number bases.